### DISCUSSION TITLE
Invisible object is behind transparent object ?

### DISCUSSION THREAD

**Discussion by ghost** [google/filament#3905] [Q&A]
Hello.
I have two 3d object.
One of them is transparent and is called A(a glass). The other is called B.
Normally, we can see B when B is behind A.
But I want to invisible any part of B if B is behind A.
So can you give me any advice ?
Thank you.

**Comment by romainguy** [comment]
You could give your object a transparent material (but draw it opaque) and give it a lower priority so it's drawn after the other transparent objects. That will work as long as the other transparent objects write to the depth buffer.

**Reply by ghost** [reply]
Thanks for your comment. I will try it.
